lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -99,12 +99,13 @@
             },
 
             saveValue: function () {
-                var attributes = {},
-                    value = this.fieldValue() ? [{'code': this.fieldValue()}] : [];
-                attributes[this.options.valueAttribute] = value;
-                this.saveAttributes(attributes);
+                if (this.persistChanges === true) {
```

Comment:
It seems to me like persisting changes has historically been the default behavior. Why change it?